scripts/deps.sh

```diff
@@ -22,10 +22,14 @@
 ctrl_c()
 {
   exit $?
-} 
+}
 trap ctrl_c SIGINT
 
 # First load snap deps
+echo "Installing dependencies if not exists"
+go get github.com/tools/godep
+go get github.com/intelsdi-x/snap/control
```

Comment:
There is an inherit assumption with the current deps.sh file that snap is located in the $GOPATH. While adding snap as a dependency to fetch before running godep on the project does solve the issue, it has the possibility to create issues for someone trying to make modifications to a snap project located outside of the $GOPATH that is then being compiled against the version located in the $GOPATH.

It would be better to update the instructions for people just looking to download and build from source to use go get github.com/intelsdi-x/snap instead of git clone. This will place snap in the $GOPATH and building of the project will work.

Justin Guidroz
Sent from my iPhone

> On Dec 3, 2015, at 9:20 AM, Joel Cooklin <notifications@github.com> wrote:
> 
> In scripts/deps.sh:
> 
> >  trap ctrl_c SIGINT
> >  
> >  # First load snap deps
> > +echo "Installing dependencies if not exists"
> > +go get github.com/tools/godep
> > +go get github.com/intelsdi-x/snap/control
> To be as clear as possible let's 'go get github.com/intelsdi-x/snap' rather than a specific package within snap.
> 
> —
> Reply to this email directly or view it on GitHub.
>